### DISCUSSION TITLE
Insigths per role and status

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#18958] [Feature Requests]
### Summary

Not every one should see every insights

### Basic Example

Certain dashboards may be under revision: they should have a `status` field
Others could be reserved for certain roles: they should have `role` field

Certain Query could vary among the roles, certain panels could be shown or not according to the roles, etc.

### Motivation

Show to the user what he expects from its point of vue.

### Detailed Design

To keep things simple just let us 

- edit the name of a dashboard
- let us use add fields (as it is the case) but give the ability to edit them underneath the dashboard.
- let us use sub-fields like `$CURRENT_ROLE.quality` in the queries (where `quality` is an added field to `directus_roles`)

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

-

### Alternatives

Create a `dashboards` collection that drives all the requirements by updating added fields to `directus_dashboards` through triggers.
But that is far from ideal. Better act directly on `directus_dashboards`.

### Adoption Strategy

easy

**Comment by Pachat** [comment]
Plus: Responsiveness would be appreciated.

**Comment by u12206050** [comment]
I use the `note` field on the dashboards to determine if a specific role can view/edit a specific dashboard. Would be nice, if we can add an additional `tags` field or something, so I don't need to use `note` field.
Have tried adding a custom field but it doesn't show in the UI
